packages/design-system-docs/src/pages/components/UsaBanner/usabanner.example.html

```diff
@@ -6,7 +6,16 @@
       class="ds-c-usa-banner__header-flag"
       xmlns="http://www.w3.org/2000/svg"
     >
-      <use xlink:href="{{root}}/dist/images/usa-banner-flag.svg"></use>
+      <g fill="none" fill-rule="evenodd">
```

Comment:
Is there a reason why we cant link to the banner icon?

Reply:
The icons won't show up - https://github.com/CMSgov/design-system/pull/888#issuecomment-735895196

Reply:
@nichia Have you tried looking into why the icons arent showing up?

Reply:
I missed the cause, will check into it

Reply:
Haven't found the cause/fix yet. Adding another ticket for this.